File: fluoxetine_backstage/src/main/resources/static/assets/js/service/noteTypeService.js
app.service('noteTypeService',function ($http) {

    this.search = function (entity,page,size) {
      return $http.post('/nodeType/search?page='+page+'&size='+size,entity);
    };

    this.add = function (entity) {
        return $http.post('/nodeType/add',entity);
    };

    this.findOne = function (id) {
      return $http.get('/nodeType/findOne?id='+id);
    };

    this.update = function (entity) {
        return $http.post('/nodeType/update',entity);
    };

    this.delete = function (ids) {
      return  $http.get('/nodeType/delete?ids='+ids);
    };

    this.findAll = function () {
        return $http.get('/nodeType/findAll');
    }

});

File: fluoxetine_backstage/src/main/java/cn/fluoxetine/fluoxetinebackstage/service/NoteTypeService.java
package cn.fluoxetine.fluoxetinebackstage.service;

import cn.fluoxetine.fluoxetinebackstage.pojo.FtbNoteType;
import cn.fluoxetine.fluoxetinebackstage.pojo.PageResult;

import java.util.List;

/**
 * @author 11699
 * @date 2020/2/10 - 13:09
 */
public interface NoteTypeService {

    public PageResult search(FtbNoteType noteType,int pageNum,int pageSize);

    public void add(FtbNoteType noteType) throws Exception;

    public FtbNoteType findOne(Integer id);

    public void update(FtbNoteType noteType);

    public void delete(Integer[] ids);

    public List<FtbNoteType> findAll();
}
